content/docs/reference/cookies.mdx

```diff
@@ -180,38 +186,17 @@ This setting cannot be set to `false` if [**Cookie SameSite**](#cookie-samesite)
 <Tabs>
 <TabItem value="Core" label="Core">
 
-| **Config file keys** | **Environment variables** | **Type**  | **Default** |
```

Comment:
I don't think it makes sense to remove the setting reference here in the v0.25 docs.